tests/infra_test.go

```diff
@@ -683,6 +684,9 @@ var _ = Describe("Infrastructure", func() {
 			for ix := 0; ix < concurrency; ix++ {
 				err := <-errors
 				if err != nil {
+					if urlErr, ok := err.(*netURL.Error); ok {
```

Comment:
What are the expected errors? I think it's better to verify we get exactly the expected errors than excluding only the connectivity error (I guess it is not the only error that have to be excluded).

Reply:
This is a specific error that i think we should have a special alert about,

the only other error i saw, which is "legit" is this one
```
client: request: &{GET https://10.244.140.74:8443/metrics HTTP/1.1 1 1 map[] <nil> <nil> 0 [] false 10.244.140.74:8443 map[] map[] <nil> map[]   <nil> <nil> <nil> 0xc0000de010} #0: Get https://10.244.140.74:8443/metrics: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
```

Reply:
Added the good path assert